transformations/dev-prod-mode/dev-prod-mode.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Development/Production Mode Tutorial
+permalink: /transformations/dev-prod-mode/
+---
+
+* TOC
+{:toc}
+
+The feature Development/Production Mode enables you to develop code above your production data without affecting
+or endangering the data in any way. It works as a version system where you create a new branch, make changes 
+in it, and run your configurations as needed. Output data is stored separately and does not overwrite your
+production data. 
+
+In this tutorial, we will show you how Development/Production Mode works, what the process of developing in 
+a branch looks like, and what you need to do before you can start.
+
+## Private Beta Warning
+Development/Production Mode is currently in private beta. Some features may not work as expected. Please bear 
+with us while we polish all necessities. Any feedback is welcome at ideas@keboola.com.
+
+## Before You Start
+Before you start using Development/Production Mode, you need to create a testing project under 
+the [Free Plan](/management/payg-project/) (available on our [website](https://www.keboola.com/)). 
+Send your request for testing this beta feature to Keboola Support with the project URL/ID, for example,
+`https://connection.north-europe.azure.keboola.com/admin/projects/1255/dashboard`, and we will give 
+you access.
+
+In the next steps, we will create and configure a transformation to demonstrate the modes on.
+
+###  Create Transformation
+In your testing project, create a new **Python** transformation and name it, for instance, *Testing dev/prod mode*. 
+
+{: .image-popup}
+![Screenshot - Create Transformation](/transformations/dev-prod-mode/01-new-transformation.png)
+
+### Create Shared Code
+Then go back to **Transformations** and prepare the following [shared code](/transformations/variables/#shared-code) 
+in Python for the transformation. Name it *Create a file*. 
+
+*Name:* `Create a file`
+<br> *Type:* `Python`
+<br> *Shared code:*
+{% highlight sql %}
+f = open("out/files/{{outFile}}", "a")
+f.write("{{outFileContent}}")
+f.close ()
+{% endhighlight %}
+
+Finally, enter two out variables `outFile`, `outFileContent`, and click **CREATE SHARED CODE**. 
```

Comment:
to ale verzálkama nikde nepíšem ne?

Reply:
jj, treba tady https://help.keboola.com/transformations/variables/#example-shared-code-with-variables

Reply:
Nevidím :) 

![image](https://user-images.githubusercontent.com/4319320/108762815-d8333480-7550-11eb-96f1-12bf21066e2f.png)

Reply:
Asi koukas na neco jinyho:

![image](https://user-images.githubusercontent.com/16991227/108815434-c2eff180-75b4-11eb-97f9-d5ebd465f79e.png)

na https://help.keboola.com/transformations/variables/#example-shared-code-with-variables

Reply:
Aha, no to se mě sice taky úplně neliíbí (imho tam normálně dáváme bold), ale myslel jsem tohle

![image](https://user-images.githubusercontent.com/4319320/108827269-b1631580-75c5-11eb-88d9-c8ea85872a47.png)

Reply:
jasne, uz chapu, co myslis. Mas pravdu, zejtra to opravim.